encoder/src/main/java/openlr/encoder/routesearch/RouteSearch.java

```diff
@@ -134,7 +131,7 @@ public RouteSearch(final List<? extends Line> loc)
         endLoopIndex = checkLoopAtEnd();
     }
 
-    /**
+    /**ß
```

Comment:
I think you inserted a character here by accident.

Reply:
d2e779368ba0835d17385a622cc639523649e302